### ISSUE TITLE
Interview completed script

### ISSUE THREAD

**Issue by C-Love** [MN-Script-Team/DHS-MAXIS-Scripts#1234]
Create a script that can case note when an interview has been completed but STAT panels have not been completely updated. It would case note some essential portions of an interview and allow workers to note things. 

The NOTES - CAF script could then be run later to fully document the CAF processing. 

This is particularly useful when a worker has interviews stacked closely together and does not have time to update STAT between each interview - thus using the CAF script directly after the interview inefficient.

**Comment by IlseFerris** [comment]
What about having an option in the CAF status be "Interview complete, processing not complete" (or something like that).  My county struggles already with "what script to use when", as I'm sure others do too.  Thoughts?  I'm trying to hackathon remotely here as best as I can :)

**Comment by CDPotter** [comment]
@IlseFerris  I would say that may not be the best idea. Say you run the CAF on a 3 year old case, it's gonna pull through any old panel that's still out there case noting old information. Seems like it would clutter up case notes to run the CAF twice.

I think I like the idea as something simple but it's hard to think of what should be required and what shouldn't. 

To me an interview completed script would probably best be based on something like worker interview form? (not sure if that's anoka specific or not) I could foresee it having basic things like enter the HH comp (2A 3C) enter programs applying for, questions answered one way on the CAF but another in the interview (though that may be defeating the purpose of the quick interview completed note).

**Comment by theVKC** [comment]
@IlseFerris: I agree with @CDPotter's interpretation. That's precisely why such a script makes sense.

@CDPotter: I don't think the WIF is currently in use, but it was not county specific; that was a DHS form.

**Comment by theVKC** [comment]
@C-Love, this script has my support. Go for it! :)

**Comment by IlseFerris** [comment]
Makes sense.  My initial thought was information that was pertinent could stay in the case note, rest could be redacted or updated manually to reflect what is known/updated at intake. Just wanted to throw my two cents in. Remember how I'm not there, sigh.....
I thought of a WIIF case note myself at one point, but the WIIF was been erased from all existence. I like this, it will really help with our business processes here at the HC.

**Comment by C-Love** [comment]
I ended up with a pretty simple script. I tried to keep it something workers can run very quickly but still give some basic information. I have kind of highlighted the XFS portion as that seems to be the most urgent needs when interviews are completed on SNAP applications.

Other than HH Comp, it does not pull any information from STAT to autofill - so workers do not need STAT to be completely up to date.
